components/form-fields/issues.jsx

```diff
@@ -10,7 +10,7 @@ class IssuesField extends React.Component {
       selectedOptions: this.props.value
     };
   }
-  componentWillMount() {
```

Comment:
Swapped 'componentWillMount' with 'componentDidMount' as recommended by React

Reply:
Same reason.

We can't move `Service.issues.get()...` to `constructor` as the call is async and `this.state = { ... }` needs to be set synchronously within `constructor`.

Reply:
What I said above is not true. Because

> Service.get() immediately returns a value (namely, a Promise) and code execution continues, so the constructor doesn't wait for Service.issues.get().then(...) to finish.

I've updated the PR. 🙈